### DISCUSSION TITLE
我在使用代理的时候请求出错，我是用nginx做的代理请帮我看一下.

### DISCUSSION THREAD

**Discussion by Soul-Journey** [betalgo/openai#319] [Q&A]
这是我的代码
`builder.Services.AddOpenAIService(settings => { settings.ApiKey = AppSettings.AI.apiKey; })
 .ConfigurePrimaryHttpMessageHandler(() => new HttpClientHandler
 {
     Proxy = new WebProxy(AppSettings.AI.Proxy),
     UseProxy = true
 }
);`

这是我nginx代理
`location /v1/ {
		proxy_pass  https://api.openai.com/v1/;

    		proxy_http_version 1.1;
    		chunked_transfer_encoding on;
   		proxy_max_temp_file_size 0;
    		proxy_buffering off;
    		proxy_cache off;

    		proxy_set_header Connection '';
    		proxy_set_header Host api.openai.com;
    		proxy_set_header content-type "application/json";
    		proxy_set_header X-Forwarded-For $remote_addr;
    		proxy_set_header X-Forwarded-Proto $scheme;
    
    		proxy_connect_timeout              15s;
    		proxy_send_timeout                 15s;
    		proxy_read_timeout                 15s;
	}`

调用是  var modelList = await _openAIService.Models.ListModel();
错误是The proxy tunnel request to proxy '' failed with status code '400'.\
但是我用postman调用没有问题

**Comment by WeihanLi** [comment]
可以使用 fiddler 抓一下 http client 的 request，看一下和 postman 的差异

**Comment by ScriptSorcererUK** [comment]
I think you may get more replies/comments on this if you also add an English translation to your post title.
